test_unstructured/partition/test_email.py

```diff
@@ -489,11 +489,10 @@ def test_partition_email_can_process_min_max_with_attachments(
     assert elements[0].text.startswith("Hello!")
     assert elements[-1].text == attachment_elements[-1].text
     assert elements[-2].text == attachment_elements[-2].text
-    email_attachments = []
     for element in elements:
         if element.metadata.attached_to_filename is not None:
-            email_attachments.append(element)
-    assert len(email_attachments) == len(attachment_elements)
+            assert len(element.text) <= 13
```

Comment:
does this actually have to be 13 and not 12?

Reply:
No, not sure why I did that in the moment. I do think that the min/pax partition logic can return a last element larger than the max param, though, if in the iteration the last item is smaller than the min partition (that is, if the last item is smaller than the min partition, it gets appended to the prior item, so the combined result might be larger than the max param). Doesn't seem to be the case here, though.